src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -88,7 +89,7 @@ public ProjectInformation resolve(@Nullable String projectUrl, @Nullable String
     }
   }
 
-  private static void getCETask(WsClient wsClient, ProjectInformation projectInfo, @Nullable String ceTaskId) throws Exception {
+  private static void getCETask(WsClient wsClient, ProjectInformation projectInfo, String ceTaskId) throws Exception {
     if (ceTaskId == null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)